buildme_deps

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+source /etc/profile.d/z00_lmod.sh
+module load gcc/8.1.0
+#module load gcc/7.1.0
+#module load cmake/3.9.2
+
+set -x
+
+installdir=`pwd`/install
+
+mkdir install
+
+mkdir deps
+cd deps
+
+depsdir=`pwd`
+
+if [ ! -d KVTree ] ; then
+  git clone git@github.com:ECP-Veloc/KVTree.git KVTree
```

Comment:
Let's use ``git clone --depth 1`` in all git clone commands if we can.  That can drastically reduce the amount of data and time, especially on repos with long histories.